examples/pytorch/tgn/dataloading.py

```diff
@@ -276,7 +276,7 @@ def __init__(self, g, eids, graph_sampler, device='cpu', collator=TemporalEdgeCo
         self.collator = collator(g, eids, graph_sampler, **collator_kwargs)
 
         assert not isinstance(g, dgl.distributed.DistGraph), \
-            'EdgeDataLoader does not support DistGraph for now. ' \
+            'DataLoader does not support DistGraph for now. ' \
```

Comment:
nit: avoid using \ to change newline. Please update since this PR touched this line :)

Reply:
sure, changed for fix.